lib/module_importer.js

```diff
@@ -32,8 +32,17 @@ function readFirstFile(uri, possibleNames, filenames, cb) {
 }
 
 // This is a bootstrap function for calling readFirstFile.
-function readAbstractFile(uri, origin, file, cb) {
-  var possibleNames = efs.expandFileName(uri, origin, file);
+function readAbstractFile(uri, origin, location, includePaths, cb) {
+  var possibleNames = efs.expandFileName(uri, origin, location),
+    thisLocation;
+
+  // Useful for further extension/debugging of the algorithm:
+  // console.log("Note: uri, origin, location: ", uri, origin, location);
+
+  for (var i in includePaths) {
+    thisLocation = path.resolve(location, includePaths[i]);
+    possibleNames = possibleNames.concat(efs.expandFileName(uri, origin, thisLocation));
+  }
```

Comment:
Will do. My JavaScript has a foot in the past! I’ve always been coding for maximum browser compatibility, but this is node — and now Babel is a thing. Gotta rewire my brain :D